### ISSUE TITLE
Add Watch Count Threshold

### ISSUE THREAD

**Issue by daniellmb** [kentcdodds/ng-stats#41]
It'd be nice to have a similar colorized threshold setting for watch count as there is for digest time, with a best practice default included. Some example implementation snippets:

```js
// defaults ...
digestTimeThreshold: 16,
watchCountThreshold: 2000,
// ...
```
```js
// color log ...
if (thingToLog === 'digest') {
  color = tracked > opts.digestTimeThreshold ? 'color:red' : 'color:green';
} else if (thingToLog === 'watches') {
  color = tracked > opts.watchCountThreshold ? 'color:red' : 'color:green';  
}
// ...
```
```js
// add data to canvas ...
var color = (averageDigest > opts.digestTimeThreshold || lastWatchCount > opts.watchCountThreshold) ? 'red' : 'green';
// ...
```
I'd be happy to submit a PR :)

**Comment by kentcdodds** [comment]
Sounds good. Though if this is done, I'd like it if the canvas differentiated the digest and watch count with regards to color. Right now the watch count will turn red if the digest is too high. I'd like them to be independent if this were implemented.

**Comment by kentcdodds** [comment]
Released in `2.4.0`. Thanks!
